packages/quantic/force-app/main/default/lwc/quanticCardContainer/quanticCardContainer.css

```diff
@@ -7,4 +7,10 @@
     background-clip: padding-box;
     -webkit-box-shadow: 0 2px 2px 0 rgb(0 0 0 / 10%);
     box-shadow: 0 2px 2px 0 rgb(0 0 0 / 10%);
+}
+
+.card__header{
+    display: flex;
+    cursor: pointer;
+    justify-content: space-between;
```

Comment:
no, this is will give an equivalent display of the QuanticCardContainer, the only change is that the cursor will be a pointer when the card container header is hovered since this element is now clickable and used to open and close the card.